lib/app.js

```diff
@@ -66,23 +65,6 @@ class Container extends Component {
 function createUrl (router) {
   return {
     query: router.query,
-    pathname: router.pathname,
-    back: () => router.back(),
```

Comment:
I've remove all of the methods from the `url` prop. We can do these with `next/router` and standard browser tools.